.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+* @silinternational/go-devs
+/terraform @silinternational/tf-devs
```

Comment:
Cool... I didn't know we could do folder-specific code-owners. 👍

Reply:
Yep, even file-level entries work. Like `*.go @silinternational/go-devs`